### PR TITLE
Add unit tests for exisitng components

### PR DESCRIPTION
**Pull request by kgaikwad**
+ adding @karelhala for review

### CODE REVIEW

**Comment by kgaikwad** [comment]
Tests are green now :) 
Fixed the snapshot tests in ProgressScoreBar.test.js

**Comment by kgaikwad** [comment]
Hello @karelhala & @apuntamb, 
Please request to review and let me know if we can merge these initial set of unit tests.

File: src/Components/RecommendationRating/RecommendationRating.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { cleanup, render } from '@testing-library/react';
+import RecommendationRating from './RecommendationRating';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+it ('matches snapshot with positive feedback', ()=>{
```
Comment (karelhala) [inline src/Components/RecommendationRating/RecommendationRating.test.js:9]: It's better to wrap test cases in `describe` for instance these could be wrapper in `RecommendationRating`. It's not a mistake, but it wraps a few tests in one nice group so you can see rather quickly what failed and which component you need to look at.

**Review by karelhala — APPROVED** [review]
Looking good! Just FYI be careful with snapshot testing, it's a good tool, however it can hurt you that generally folks tend to overlook them and merge right away. It is good to join together query selectors and snapshot testing. That way you know some crucial element is present and the snapshot is for all the details around these elements.

File: src/Components/RecommendationRating/RecommendationRating.test.js (hunk 1 above)
Comment (kgaikwad) [reply src/Components/RecommendationRating/RecommendationRating.test.js]: Thank you @karelhala for review. Added test cases inside `describe` and also tried to add query selectors inside snapshot. 
Please have a look at new changes & suggest if any additional changes required.

**Review by karelhala — APPROVED** [review]
Really good approach!

**Comment by kgaikwad** [comment]
Thank you @karelhala! Merging the changes.

**Pull request merged**